### ISSUE TITLE
Impossible to add an article in backend with French language in 3.4.4 as of 08/13/15

### ISSUE THREAD

**Issue by Pachat** [joomla/joomla-cms#7691]
#### Steps to reproduce the issue
1- Download Joomla from GitHub
2- Install French language (from Language Install or [from Joomla.fr](http://aide.joomla.fr/telechargements/joomla-3-x-package-d-installation-et-patchs/fichiers-de-langue-pour-joomla-3-x/fichier-de-langue-complet-administration-et-site-3-4-3-v1/download "Fichier de langue complet (administration et site) 3.4.3 v1")
In the backend, Add New Article
3- No buttons (even Cancel) work.

**Comment by infograf768** [comment]
Adding a language to your installation has no impact on creating an article.
I can't reproduce your issue.

**Comment by justinherrin** [comment]
I have experienced the "no buttons work" thing before and a simple clear of the browser cache has fixed it for me each time.

**Comment by Devportobello** [comment]
Pretty sure this is same as https://github.com/joomla/joomla-cms/issues/7880

**Comment by infograf768** [comment]
Nope. Adding a new article does not load the modal for which we got the single quote error

**Comment by diglin** [comment]
Please check this issue, it's similar with more details : [http://issues.joomla.org/tracker/joomla-cms/7897](http://issues.joomla.org/tracker/joomla-cms/7897 "http://issues.joomla.org/tracker/joomla-cms/7897")<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7691">issues.joomla.org/joomla-cms/7691</a>.</sub>

**Comment by infograf768** [comment]
If it is the same issue, then it is solved in https://github.com/joomla/joomla-cms/pull/7845

Please update your French language pack.
